index.html

```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
         
+        <!-- ________ *ALCUNOO* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">ALCUNO</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
```

Comment:
Since you are mentioning your github so icon should also be changed to github icon.
So please replace `fab fa-twitter` with `fab fa-github`.